README.md

```diff
@@ -48,34 +45,33 @@ With Mbed OS 5.9, the EMAC SW was refactored and a default network selector is u
 
 ### Other WiFi stacks
 
+Options for "network-interface" are ETHERNET, WIFI_ESP8266, WIFI_IDW0XX1, WIFI_ODIN, WIFI_RTW, WIFI_WIZFI310, WIFI_ISM43362, MESH_LOWPAN_ND, MESH_THREAD, CELLULAR_ONBOARD
 If you select `WIFI_ESP8266`, `WIFI_IDW0XX1`, `WIFI_ODIN` or `WIFI_RTW`, `WIFI_WIZFI310` you also need to add the WiFi SSID and password:
 
 ```json
-    "config": {
-        "network-interface":{
-            "help": "options are ETHERNET, WIFI_ESP8266, WIFI_IDW0XX1, WIFI_ODIN, WIFI_RTW, WIFI_WIZFI310, WIFI_ISM43362, MESH_LOWPAN_ND, MESH_THREAD, CELLULAR_ONBOARD",
-            "value": "WIFI_ESP8266"
-        },
-        "wifi-ssid": {
-            "value": "\"SSID\""
-        },
-        "wifi-password": {
-            "value": "\"Password\""
+    "target_overrides": {
+        "*": {
+            "easy-connect.wifi-type": "\"ESP8266\"",
```

Comment:
Seems like duplication that you have to tell it so many times, it's ESP8266? Doesn't seem right to me. One selection MUST be enough and then rest of it needs to happen outside of the `mbed_app.json`.